Makefile

```diff
@@ -299,7 +300,7 @@ git-status:
 	git status --porcelain
 
 git-commit:
-	git diff-index --quiet HEAD || git commit -m "Semaphore Automatic Update" --author "Semaphore Automatic Update <marvin@tigera.io>" go.mod go.sum
+	git diff-index --quiet HEAD || git commit -m "Semaphore Automatic Update" -c user.name="Semaphore Automatic Update" -c user.email="<marvin@tigera.io>" go.mod go.sum
```

Comment:
What difference does this make?

Reply:
Turns out git requires a user to be configured, which then can specify a different author. But the former is still needed.

Reply:
I think it's worth documenting that, so that it doesn't get reverted, when some other motivation crops up, just because someone didn't understand why it was there.  In order of preference:

1. In a comment just above that line.
2. In the Git commit message.
3. In the PR description.

WDYT?

Reply:
Don't really need a comment, the tests will fail in CI if someone removes these options. It passed on my machine because I already had these configured in my git config file.

Reply:
Ah, OK then.  Thanks for explaining!